hles/transformation/src/test/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformationsSpec.scala

```diff
@@ -532,5 +548,7 @@ class DemographicsTransformationsSpec extends AnyFlatSpec with Matchers with Opt
     assistanceOut.ddActivitiesServiceOther.value shouldBe true
     assistanceOut.ddActivitiesServiceOtherMedicalDescription.value shouldBe "Medical!"
     assistanceOut.ddActivitiesServiceOtherDescription.value shouldBe "Activity!"
+
+    activitiesOut.ddActivitiesCompanionAnimal.value shouldBe 1L
```

Comment:
i think this spec is too big. when you test too many things like this, it becomes hard to identify what part of the service isn't working properly when the spec fails. can we split out the fallback behavior added in this PR into its own test?